part-spec/semiconductor.json

```diff
@@ -0,0 +1,277 @@
+                },
+                "mpn": {
+                    "description": "manufacturer part number",
+                    "type": "string"
+                },
+                "datasheetVersion": {
+                    "description": "datasheet version encoded in this document",
+                    "type": "string"
+                },
+                "eolDate": {
+                    "description": "date this part will no longer be supported by manufacturer",
+                    "type": "string",
+                    "format": "date"
+                },
+                "type": {
+                    "description": "type of transistor",
+                    "examples": [
+                        "mosfet",
+                        "bjt",
+                        "jfet",
+                        "etc."
+                    ],
+                    "type": "string"
+                },
+                "channel": {
+                    "description": "doping of a transistor's channel - describes whether a transistor is n-type or p-type",
+                    "enum": [
+                        "nType",
+                        "pType"
+                    ],
+                    "comment": "units of string"
+                },
+                "vgsMax": {
+                    "description": "maximum gate to source voltage difference that can be continously applied to a mosfet",
+                    "comment": "units of volts",
+                    "type": "number"
+                },
+                "vdsMax": {
+                    "description": "maximum drain to source voltage difference that can be continously applied to a mosfet",
+                    "comment": "units of volts",
+                    "type": "number"
+                },
+                "vgsThMax": {
+                    "description": "maximum gate to source voltage difference required to produce a conducting path between drain and source",
+                    "comment": "units of volts",
+                    "type": "number"
+                },
+                "iDrain": {
+                    "description": "maximum continous DC current that can flow through a mosfet channel",
+                    "comment": "units of amps",
+                    "type": "number"
+                },
+                "idPulsed": {
+                    "description": "maximum pulsed DC current that can flow through a mosfet channel",
+                    "comment": "units of amps",
+                    "type": "number"
+                },
+                "rdson": {
+                    "description": "drain source on resistance of a mosfet",
+                    "comment": "units of ohms",
+                    "type": "number"
+                },
+                "rdsonCondition": {
+                    "description": "description of the gate-source voltage and drain current under which rdson is measured",
+                    "examples": [
+                        "\"vgs = 2.5V",
```

Comment:
what are the \ for?

Reply:
huh i'm not sure... must have been a translation artifact. I'll check the MD and resolve that.